src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/MuTect2.java

```diff
@@ -954,23 +908,6 @@ private Haplotype createReferenceHaplotype(final AssemblyRegion assemblyRegion,
         return ReferenceConfidenceModel.createReferenceHaplotype(assemblyRegion, assemblyRegion.getAssemblyRegionReference(referenceReader), paddedReferenceLoc);
     }
 
-    /**
-     * Clean up reads/bases that overlap within read pairs
-     *
-     * @param reads the list of reads to consider
-     */
-    private void cleanOverlappingReadPairs(final List<GATKRead> reads, final Set<String> normalSampleNames) {
```

Comment:
This method was not used in GATK3 M2, but GATK4 HC does. Might be useful to keep it (and find the right place to use it).